outline/shadowsocks/client.go

```diff
@@ -29,20 +34,15 @@ import (
 	"github.com/eycorsican/go-tun2socks/common/log"
 )
 
-// Config represents a shadowsocks server configuration.
-// Exported via gobind.
-type Config struct {
-	Host       string
-	Port       int
-	Password   string
-	CipherName string
-	Prefix     []byte
-}
-
+// [Exported] A Shadowsocks client that can be used by Outline-Apps
 type Client = outline.Client
 
-// NewClient provides a gobind-compatible wrapper for [client.NewClient].
+// [Exported] Create a new Shadowsocks client from a non-nil configuration.
```

Comment:
Go has a very specific style for comments that started with the function name. We should keep following that.